pkg/validation/release_test.go

```diff
@@ -155,7 +156,7 @@ func TestValidateReleases(t *testing.T) {
 			},
 		},
 		{
-			name: "invalid name of integration release",
+			name: "invalid release name",
```

Comment:
Add some cases here covering the other types that are now validated, just for completeness.

Reply:
Done.